content/en/tracing/trace_collection/dd_libraries/php.md

```diff
@@ -60,11 +60,20 @@ Install and configure the Datadog Agent to receive traces from your now instrume
 {{< partial name="apm/apm-containers.html" >}}
 </br>
 
-3. After having instrumented your application, the tracing client sends traces to `localhost:8126` by default.  If this is not the correct host and port change it by setting the below env variables:
+3. After having instrumented your application, the tracing client sends traces to Unix domain socket `/var/run/datadog/apm.socket` by default. If the socket does not exist, traces are sent to `http://localhost:8126`.  If this is not the correct host and port change it by setting the below env variables:
 
-    `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT`.
+    `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` or `DD_TRACE_AGENT_URL`.
```

Comment:
NItPicking - I think we should advertize `DD_TRACE_AGENT_URL` mainly (so maybe putting it first, even removing the others?)

Reply:
done